schema/atac_schema.md

```diff
@@ -0,0 +1,473 @@
+## scATAC-seq assay types
+
+<i>paired assay</i>: any descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0010891"><code>"EFO:0010891"</code></a> for <i>scATAC-seq</i> that is also a descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008913"><code>"EFO:0008913"</code></a> for <i>single-cell RNA sequencing</i>
+
+<i>unpaired assay</i>: <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0010891"><code>"EFO:0010891"</code></a> for <i>scATAC-seq</i> or its descendants that is not a descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008913"><code>"EFO:0008913"</code></a> for <i>single-cell RNA sequencing</i>
+
+## Fragment File Dataset Criteria
+
+A Dataset MUST meet each of the following criteria in order to be eligible for an attached Fragment File:
+* the <code>obs['assay_ontology_term_id']</code> values MUST all be <i>paired assays</i> or MUST all be <i>unpaired assays</i>
+* the <code>obs['is_primary_data']</code> values MUST be all `True`
+* the <code>var['feature_reference']</code> values MUST include one of <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i> or <code>"NCBITaxon:10090"</code> for <i>Mus musculus</i>, but not both. The value that is present will determine the appropriate Chromosome Table for standards.
+
+If the <code>obs['assay_ontology_term_id']</code> values are all <i>paired assays</i> then a fragment file MAY be attached to the Dataset.
+
+If the <code>obs['assay_ontology_term_id']</code> values are all <i>unpaired assays</i> then a fragment file MUST be attached to the Dataset.
+
+## Fragment File
+
+This MUST be a gzipped tab-separated values (TSV) file.
+
```

Comment:
This section is a minor _variation_ of [Column Definitions](https://www.10xgenomics.com/support/software/cell-ranger-arc/latest/analysis/outputs/fragments-file#column-definitions) for the most part?

Reply:
Is there a standard name for the fragment file?

Reply:
>This section is a minor variation of Column Definitions for the most part?

Correct

>Is there a standard name for the fragment file?

As in _file_ name? Similar to what we see with the `.h5` raw matrices, it's the pipeline output file amended somehow with a library/sample name to distinguish each file in their study.
So no.

Reply:
Depending on the final submission design, we MAY want to standardize the naming. For example, `<dataset canonical UUID>-fragments.tsv.gz`.

Reply:
I added the naming convention to the updated version